multiapi-engine/src/test/java/com/sngular/api/generator/plugin/asyncapi/AsyncApiGeneratorFixtures.java

```diff
@@ -214,6 +289,8 @@ static Function<Path, Boolean> validateTestFileGenerationIssue() {
 
     final String DEFAULT_MODEL_FOLDER = "generated/com/sngular/scsplugin/filegenerationissue/model/event";
 
+    final String DEFAULT_CUSTOMVALIDATOR_FOLDER = "generated/com/sngular/scsplugin/filegenerationissue/model/event/customvalidator";
```

Comment:
Does this constant alaready been created in L.242 (and the same for L.290)?
Could it be possible (has sense) to use the first one instead of duplicating them?

Reply:
Done